build.sbt

```diff
@@ -58,9 +58,9 @@ lazy val plugin: Project =
   (project in file("plugin")) settings (
     sharedSettings: _*
   ) settings (
-    addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0-M5" cross CrossVersion.full),
+    addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
     libraryDependencies <+= (scalaVersion)("org.scala-lang" % "scala-compiler" % _ % "provided"),
-    libraryDependencies += "org.scalameta" % "scalahost" % "0.1.0-SNAPSHOT" cross CrossVersion.full,
+    libraryDependencies += "org.scalameta" % "scalahost_2.11.8" % "1.6.0",
```

Comment:
In this case, scalahost needs the full compiler version (2.11.8) instead of just major.minor (2.11) like most cross-compiled Scala jars.  If there is a way to tell sbt to use a variable (like scalaVersion in this build.sbt file) or to use the full compiler version, I would love to hear about it.  However, I don't know about such a technique.